build_local_image.sh

```diff
@@ -37,27 +37,31 @@ function check_usage {
 }
 
 function main {
+	check_utils rsync
```

Comment:
Please move this check to the already existing **check_utils** call in the **check_usage** function.